src/collective/cover/interfaces.py

```diff
@@ -30,14 +32,8 @@ def getSearchResults(self, filter_portal_types, rooted, document_base_url, searc
 
 
 class ICoverUIDsProvider(Interface):
-
-    def getUIDs(self):
-        """Get UUIDs associated with the object.
-            could be the UUID of the object or a
-            list of related UUIDs.
-
-        @return: iterable of UUIDs
-        """
+    """GenericUIDsProvider interface will be removed in collective.cover v1.7."""
+    warnings.warn(__doc__, DeprecationWarning)
```

Comment:
The error is indeed because of the ZCML registration, but we need the provided funcionality: changing from "Could not adapt" to uuids always being `None` doesn't change much, since I could just do a `try: except TypeError`, set uuids to None and don't bother you in this PR anymore.

As we said, we don't mind removing these adapters, but they should be removed in 1.7 as is shown at the `warnings` message. We think that the deprecation implementation the way it is right now isn't right.